contributing/development/core_and_modules/custom_platform_ports.rst

```diff
@@ -169,7 +169,7 @@ you have two options:
 Distributing a custom platform port
 -----------------------------------
 
-.. warning::
+.. danger::
```

Comment:
Based on the same standard of potential legal harm used to change this warning, the warning in Complying with licenses could also be a danger:
https://github.com/godotengine/godot-docs/blob/905b1f4f0805894d1a09927e384eb234ae243efc/about/complying_with_licenses.rst?plain=1#L8-L12
However, that one occurs early in the docs in the About section, so a danger might scare people off. And I think the potential harm of getting the MIT license a little wrong is less than the potential harm of distributing an NDA console port 🤷